destinations/sift/handler.js

```diff
@@ -0,0 +1,145 @@
+// Segment & Sift
+const SEGMENT_WRITE_KEY = '';
```

Comment:
Should this be a setting too?

Reply:
I think I had copied that over from another destination library/jotted down in case - but I can't seem to find my logic anymore. 

@juhaelee or @shyaankhan - can you shed some light?

Reply:
Usually would prefer these to be settings so it's not hardcoded here- and so it's easy to configure when users install the destinations

Reply:
Gotcha - @kellymtrinh , I can add a note about it in the instructions to set it as a settings key. Thanks!

Reply:
This would require a code update as well to read from settings (passed to onTrack for example) instead of the SEGMENT_WRITE_KEY variable